mongodb_migrations/base.py

```diff
@@ -5,16 +5,21 @@ class BaseMigration(object):
     def __init__(self,
                  host='127.0.0.1',
                  port='27017',
-                 database=None,
+                 database=None,                 
+                 auth_user=None,
```

Comment:
if you don't mind, can we change it to just `user` and same for below, just `password`

Reply:
sure :)